.pipelines/pipeline.user.windows.yml

```diff
@@ -0,0 +1,61 @@
+environment:
+  host:
+    os: 'windows'
+    flavor: 'server'
+    version: '2019'
+  runtime:
+    provider: 'appcontainer'
+    image: 'cdpxwinrs5.azurecr.io/global/vse2019/16.3.7:latest'
+    source_mode: 'link'
+
+version:
+  name: 'DefaultVersion'
+  major: 2
+  minor: 1
+  system: 'RevisionCounter'
+  exclude_commit: true
```

Comment:
Our current highest published library is 2.0.1. This configuration results in versions such as 2.1.10692, which seems as close as I know how to get to advancing from 2.0.1.